public/4.html

```diff
@@ -76,7 +76,7 @@
         <p>
           Mission accomplished. Colors get stored and loaded again.
           And the best thing: the colors are stored in your browser. You can
-          turn off your wifi connection, it will still work. No errors, no warnings, no different interface. It just works.
+          turn off your wifi connection, it will still work. No errors, no warnings, no different interface. It works, all the way fine.
```

Comment:
I’m not sure if that is a better expression than the current one we have? I’ll see what others think